packages/admin-panel-server/src/app/createApp.ts

```diff
@@ -11,7 +11,7 @@ import {
 
 import { AdminPanelSessionModel } from '../models';
 import { hasTupaiaAdminPanelAccess } from '../utils';
-import { upload } from '../middleware';
+import { upload, verifyBESAdminAccess } from '../middleware';
```

Comment:
I saw that this util was removed in https://github.com/beyondessential/tupaia/pull/4229. I re-instated it because if I'm not mistaken we need it for the two `entityHierarchyTree` endpoints that I have added in `admin-panel-server` for this feature.

Let me know if auth should be handled in a another way now @rohan-bes

Reply:
By the way I didn't get any merge conflicts when I merged `dev` in the base branch, although in `dev` this util has been removed and my branch uses it :confused: 

I'm not quite sure why this happened...